packages/engine/lib/stateful/src/field/key.rs

```diff
@@ -5,14 +5,15 @@ use crate::{
     Error, Result,
 };
 
-pub const HIDDEN_PREFIX: &str = "_HIDDEN_";
-pub const PRIVATE_PREFIX: &str = "_PRIVATE_";
-
+// TODO: Use a struct containing the name, the scope, and the source instead?
+// TODO: OPTIM (follow-up of the above): Use string interning for faster column name lookup
```

Comment:
I don't think we should even have this TODO, as we discussed, even with string interning moving to this would add additional operations because user code will refer to fields by their name and not their key, which means we either need to generate the key or we need to do a look-up.

Reply:
That's a left-over TODO I added before we talked about this off-Github. I removed it in afb7808828eaaf3e83fd9576b173ba8538bdecfd